### ISSUE TITLE
Is this production ready?

### ISSUE THREAD

**Issue by MrBlaise** [apache/kvrocks-controller#182]
Hey!

I am considering KVRocks, however the manual cluster setup, the missing failover feature and the 1 by 1 slot migration is a big obstacle (I will often scale up and down the cluster).

Is this kvrocks controller meant to be ready for production to support these or it is still under development?

**Comment by git-hulk** [comment]
@MrBlaise Yes, it's production ready now. Will publish a formal release soon.

**Comment by spihiker** [comment]
Come on ,come on.
